.circleci/config.yml

```diff
@@ -40,7 +40,8 @@ jobs:
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
           GOMAXPROCS: "2"
-    - prometheus/check_proto
+    - prometheus/check_proto:
+        version: "3.11.4"
```

Comment:
I have set the version here.

Once merged I could have a look at making this the default.